sys/parsoid.js

```diff
@@ -440,25 +441,25 @@ PSP.getFormat = function(format, restbase, req) {
         }
     }
 
+    if (!self._okayToRerender(req)) {
```

Comment:
If we don't split the `no-cache` logic, we'll need to duplicate the 
```
var contentReq = restbase.get({
          uri: self.getBucketURI(rp, format, rp.tid)});
});
```
code. And this check is a temporary solution, so I think its better to have all it's logic self-contained.